### PR TITLE
fixes #9122 bug(nimbus): limit multi-feature experiments to 20 features

### LINKED ISSUE #9122: Multifeature experiments that contain a large number of features cannot be launched
**[Prerequisites]:**

* Have a Nimbus multifeature experiment created with ~20 features.

**[Steps to reproduce]:**

1. Open the "Summary" page of the experiment.
1. Click the "Launch to Preview" button and observe the page.

**[Expected result]:**

* The "All set! Your experiment is in Preview mode and you can test it now." message is displayed.

**[Actual result]:**

* The "Sorry, an error occurred while submitting. Please try again." message is displayed.

**[Notes]:**

* This is most likely related to the "namespace" parameter character limit, as fewer feature configurations with longer names are needed to hit the limit (around 15).
* The character limit of the “namespace“ seems to be around 250 characters.
* This issue is also reproducible when attempting to launch without preview, the error appears after the launch is approved, but the configuration is not sent to RS.
* Attached a screenshot of the issue:

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/EXP-3683)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/39715">error_occured.png</a>

### PR DESCRIPTION
**Pull request by brennie**
Because:
 - the namespace field has a character limit; and
 - adding arbitrarily many feature configs to an experiment will cause the limit to be hit

this commit:
 - increases the max length of NimbusIsolationGroup names to 2048; and
 - adds a limit of 20 features to the selected feature configs.

### CODE REVIEW

File: experimenter/experimenter/nimbus-ui/src/styles/index.scss [hunk 1]
```diff
@@ -204,6 +204,18 @@ select.form-control.is-warning {
...
+  flex-grow: 1;
+}
+
+.feature-configs-count {
+  align-self: center;
+  text-align: center;
+  margin-left: 1rem;
+  min-width: 5rem;
+}
+
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/styles/index.scss:218]: we can use bootstrap classes to achieve this and won't need to define CSS in here

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
overall looks good, just one minor comment regarding the CSS classes [bootstrap doc](https://react-bootstrap-v4.netlify.app/layout/grid/)

**Review by yashikakhurana — APPROVED** [review]
thank you @brennie this looks perfect :)

**Review by jaredlockhart — APPROVED** [review]
Yep, tested locally, looks great, works great, thnx @brennie 🎉 🎉 🎉 🎉

**Pull request merged**